README.md

```diff
@@ -1,5 +1,4 @@
 [![Build Status](https://travis-ci.com/broadinstitute/gatk.svg?branch=master)](https://travis-ci.com/broadinstitute/gatk)
-[![codecov](https://codecov.io/gh/broadinstitute/gatk/branch/master/graph/badge.svg)](https://codecov.io/gh/broadinstitute/gatk)
```

Comment:
Comment out instead of deleting

Reply:
not sure how to comment something out of the readme in a sane way.